Description: _matplotlib
Index: VTK-9.0.1/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
===================================================================
--- VTK-9.0.1.orig/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
+++ VTK-9.0.1/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
@@ -151,6 +151,10 @@ private:
    * to implement IsAvailable.
    */
   static Availability CheckMPLAvailability();
+  static void DisableMPL()
+    {
+    MPLMathTextAvailable = UNAVAILABLE;
+    };
 
   //@{
   /**
