Description: Fix FTBFS with new netcdf
Author: Anton Gladky <gladk@debian.org>
Bug-Debian: https://bugs.debian.org/1012703
Last-Update: 2022-06-19

--- vtk9.orig/ThirdParty/exodusII/vtkexodusII/src/ex_utils.c
+++ vtk9/ThirdParty/exodusII/vtkexodusII/src/ex_utils.c
@@ -1770,7 +1770,7 @@
         */
 
         /* const int NC_SZIP_EC = 4; */ /* Selects entropy coding method for szip. */
-        const int NC_SZIP_NN = 32;      /* Selects nearest neighbor coding method for szip. */
+        // const int NC_SZIP_NN = 32;      /* Selects nearest neighbor coding method for szip. */
         /* Even and between 4 and 32; typical values are 8, 10, 16, 32 */
         const int SZIP_PIXELS_PER_BLOCK =
             file->compression_level == 0 ? 32 : file->compression_level;
