Description: Do not try linking with -lsocket

--- vtk9.orig/Common/System/vtkTestSocketFeatures.cmake
+++ vtk9/Common/System/vtkTestSocketFeatures.cmake
@@ -6,9 +6,6 @@
 
 if(NOT DEFINED VTK_HAVE_GETSOCKNAME_WITH_SOCKLEN_T)
   set(VTK_GETSOCKNAME_LIBS)
-  if(VTK_HAVE_LIBSOCKET)
-    list(APPEND VTK_GETSOCKNAME_LIBS "socket")
-  endif()
   message(STATUS "Checking for getsockname with socklen_t")
   try_compile(VTK_HAVE_GETSOCKNAME_WITH_SOCKLEN_T_COMPILED
     ${CMAKE_CURRENT_BINARY_DIR}/CMakeTmp/SocklenT
