Description: Drop medical examples
Last-Update: 2020-10-21

--- vtk9.orig/Examples/CMakeLists.txt
+++ vtk9/Examples/CMakeLists.txt
@@ -56,7 +56,7 @@
   add_example(IO/Cxx)
   add_example(ImageProcessing/Cxx)
   add_example(Infovis/Cxx)
-  add_example(Medical/Cxx)
+  #add_example(Medical/Cxx)
   add_example(Modelling/Cxx)
   add_example(MultiBlock/Cxx)
   add_example(ParallelProcessing/Generic/Cxx)
