Description: Drop medical examples
Last-Update: 2020-10-21

--- vtk9-9.0.1+dfsg1.orig/Examples/CMakeLists.txt
+++ vtk9-9.0.1+dfsg1/Examples/CMakeLists.txt
@@ -55,7 +55,7 @@ else ()
   add_example(IO/Cxx)
   add_example(ImageProcessing/Cxx)
   add_example(Infovis/Cxx)
-  add_example(Medical/Cxx)
+  #add_example(Medical/Cxx)
   add_example(Modelling/Cxx)
   add_example(MultiBlock/Cxx)
   add_example(ParallelProcessing/Generic/Cxx)
