Description: Fix path for the installation of python modules
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2020-11-06

Index: vtk9-9.0.1+dfsg1/CMake/vtkModuleWrapPython.cmake
===================================================================
--- vtk9-9.0.1+dfsg1.orig/CMake/vtkModuleWrapPython.cmake
+++ vtk9-9.0.1+dfsg1/CMake/vtkModuleWrapPython.cmake
@@ -84,7 +84,7 @@ function (vtk_module_python_default_dest
         "for Python modules.")
       set(_vtk_python_version_suffix)
     endif ()
-    set(destination "${CMAKE_INSTALL_LIBDIR}/python${_vtk_python_version_suffix}/site-packages")
+    set(destination "lib/python3/dist-packages")
   endif ()
 
   set("${var}" "${destination}" PARENT_SCOPE)
Index: vtk9-9.0.1+dfsg1/Utilities/Python/CMakeLists.txt
===================================================================
--- vtk9-9.0.1+dfsg1.orig/Utilities/Python/CMakeLists.txt
+++ vtk9-9.0.1+dfsg1/Utilities/Python/CMakeLists.txt
@@ -77,7 +77,7 @@ if (NOT VTK_PYTHON_SITE_PACKAGES_SUFFIX)
     set(VTK_PYTHON_SITE_PACKAGES_SUFFIX "Lib/site-packages")
   else ()
     set(VTK_PYTHON_SITE_PACKAGES_SUFFIX
-      "python${vtk_python_version_pair}/site-packages")
+      "python3/dist-packages")
   endif ()
 endif ()
 
