Description: Fix FTBFS against freetype 2.10.4
Author: mathstuf
Reviewed-By: Anton Gladky <gladk@debian.org>
Bug-Debian: https://bugs.debian.org/976975
Forwarded: https://github.com/Kitware/VTK/commit/3edc0de2b04ae1e100c229e592d6b9fa94f2915a
Last-Update: 2020-12-10

--- vtk9-9.0.1+dfsg1.orig/Rendering/FreeType/vtkFreeTypeTools.cxx
+++ vtk9-9.0.1+dfsg1/Rendering/FreeType/vtkFreeTypeTools.cxx
@@ -378,8 +378,7 @@ FTC_CMapCache* vtkFreeTypeTools::GetCMap
 }
 
 //----------------------------------------------------------------------------
-FT_CALLBACK_DEF(FT_Error)
-vtkFreeTypeToolsFaceRequester(
+static FT_Error vtkFreeTypeToolsFaceRequester(
   FTC_FaceID face_id, FT_Library lib, FT_Pointer request_data, FT_Face* face)
 {
 #if VTK_FTFC_DEBUG_CD
